jdi-light-bootstrap-tests/src/main/java/io/github/com/sections/PopoverBottom.java

```diff
@@ -0,0 +1,16 @@
+package io.github.com.sections;
+
+import com.epam.jdi.light.elements.composite.Section;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.bootstrap.elements.common.Popover;
+import com.epam.jdi.light.ui.bootstrap.elements.common.Text;
+import com.epam.jdi.light.ui.bootstrap.elements.complex.MediaObject;
+
+public class PopoverBottom extends Section {
+
+    @UI("#popover-bottom")
+    public Popover popover;
+    public MediaObject container;
+    public Text body;
+
+}
```

Comment:
we can create only one class instead of 4 different: PopoverBottom, .... 
it can looks like that:
```
public class Popover extends Section {
    public UIElement popoverBlock;
    public MediaObject container;
    public Text body;
    Popover(PopoverType type){
        popover=$(type.getLocator());
    }
}
enum PopoverType{
      bottom("#popover-bottom"),
      dissmissible("blabla"),
      ...
     private String locator;

     PopoverType(String locator){
           this.locator=locator;
     }
     public String getLocator(){
         return locator;
     }
}

class MyPage{ 
      Popover popoverBottom=new Popover(Popover.bottom);
      Popover popoverDissmissible=new Popover(Popover.dissmissible);
}
```